css-canvas-medical/styles.css

```diff
@@ -0,0 +1,229 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  color: rgb(100, 100, 100);
+  font-family: 'Avenir', sans-serif;
+}
+
+h1 {
+  font-family: 'Tiempos Headline', serif;
+}
+
+a {
+  text-decoration: none;
+  flex-basis: calc(100% / 8);
+  color: rgb(100, 100, 100);
+}
+
+p,
+a,
+button {
+  font-size: 1.2rem;
+  font-family: 'Avenir', sans-serif;
+}
+
+button {
+  background: #294464;
+  padding: 1.2rem 4rem;
+  border-radius: 0.4rem;
+  cursor: pointer;
+  font-size: 1.3rem;
+  outline: none;
+}
+
+.more-info-section,
+.more-info-section a,
+.hero-text-container,
+.more-info-section h1,
+button,
+.hero-text-container h1 {
+  color: rgb(238, 238, 238);
+}
+
+div.header {
+  display: flex;
+  padding: 1rem 2rem;
+  max-width: 1450px;
+  margin-left: auto;
+  margin-right: auto;
+  align-items: center;
+}
+
+.row-half {
+  flex-basis: 50%;
+}
+
+.logo img {
+  margin: 0;
+  width: 160px;
+}
+
+.links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-us-button {
+  padding: 0.6rem 1.3rem;
+}
+
+.separator {
+  border: 1px solid rgb(247, 247, 247);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.hero-section {
+  max-width: 1400px;
+  height: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  background-image: url('images/hero.jpg');
+  background-size: cover;
+  position: relative;
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+}
+
+.hero-section > img {
+  width: 100%;
+}
+
+.hero-overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.7));
+  position: absolute;
+  right: 0;
+  left: 0;
+  top: 0;
+  bottom: 0;
+}
+
+.hero-text-container {
+  position: absolute;
+  top: 5rem;
+  max-width: 650px;
+  font-family: 'Tiempos Headline', serif;
+}
+
+.hero-text-container h1 {
+  font-size: 3rem;
+}
+
+.hero-text-container p {
+  line-height: 2.3rem;
+}
+
+.hero-text-container button {
+  margin: 1.5rem 0;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 35rem;
+  border-bottom: 1px solid rgb(224, 224, 224);
+}
+
+.locations {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.location-heading {
+  max-width: 1300px;
+  max-width: 600px;
```

Comment:
Whoops, duplicated property here! The one being used is the bottom one, so you can remove the top one.